js/index.js

```diff
@@ -8,7 +8,7 @@ import {
   Clients,
   Footer,
 } from './widgets/index.js';
-import { handleLogoClick, onThemeClick } from './handlers.js';
+import { handleLogoClick, onThemeClick, onBurgerClick } from './handlers.js';
```

Comment:
```suggestion
import { 
  handleLogoClick, 
  onThemeClick,
  onBurgerClick,
} from './handlers.js';
```
Придерживайся одного стиля оформления импортов